core/platform/datastore/cloud_datastore_services.py

```diff
@@ -85,8 +86,16 @@ def get_multi(keys: List[Key]) -> List[Optional[TYPE_MODEL_SUBCLASS]]:
     Returns:
         list(datastore_services.Model | None). List whose items are either a
         Model instance or None if the corresponding key wasn't found.
+
+    Raises:
+        Exception. If ndb.get_multi fails for MAX_GET_RETRIES.
     """
-    return ndb.get_multi(keys)
+    for unused_i in range(0, MAX_GET_RETRIES):
+        try:
+            return ndb.get_multi(keys)
+        except Exception as unused_e:
+            continue
```

Comment:
I think we should still log this somehow, use smth like `logging.warning` or `logging.exception`

Reply:
Done